backend/src/openbeheer/api/drf_spectacular/schema.py

```diff
@@ -141,9 +161,27 @@ def get_schema_operation_parameters(
         component_name = out["$ref"].replace("#/components/schemas/", "")
         component_schema = components.pop(component_name)
 
+        def inline_ref(prop_schema, components):
+            # $refs should not have siblings other than "summary" or "description"
+            # https://quobix.com/vacuum/rules/schemas/oas3-no-ref-siblings/
+            #
+            # inline the ref if it does
+            # TODO: This should be fixed upstream, I think
```

Comment:
Or maybe as a post process hook... :thinking:

Reply:
Is this maybe due to the difference between JSON schema and open API?

I see [here](https://opis.io/json-schema/2.x/references.html) under `$ref`:

> In draft-06 and draft-07 sibling keywords are ignored. Starting with draft-2019-09 sibling keywords are evaluated.